Makefile

```diff
@@ -23,10 +23,15 @@ SHELL:=/usr/bin/env bash
 #
 # Go.
 #
-GO_VERSION ?= 1.22.12
+GO_VERSION ?= 1.23.8
+GO_DIRECTIVE_VERSION ?= 1.23.8
```

Comment:
Shouldn't this be kept to 1.22.0? (or not exist here)

Reply:
Thx, fixed!